### DISCUSSION TITLE
Adding ARM64 Build on releases

### DISCUSSION THREAD

**Discussion by X7md** [quickjs-ng/quickjs#417] [Q&A] [answered]
I think adding `aarch64` Linux release build will be nice, for folk's who's uses raspberry pi and so on...

**Comment by saghul** [comment, marked as answer]
Makes sense, I'll add it for the next release 👍

**Comment by KaruroChori** [comment]
Similarly, can we have the same for risc v?

**Reply by saghul** [reply]
I don't see why not!

**Reply by saghul** [reply]
Merged!
